setup.cfg

```diff
@@ -32,7 +32,7 @@ namespace_packages =
 
 [entry_points]
 console_scripts =
-    test-template = sphinxcontrib.datatemplates.cli:main
+    sphinx-datatemplate = sphinxcontrib.datatemplates.cli:main
```

Comment:
While the library depends on sphinx and got sphinx in it's name, the CLI script is not connected to sphinx directly. Just ``datatemplate`` seems better except it does not obviously belongs to ``sphinxcontrib.datatemplate``.

Reply:
Yes, you make a good point. I was looking for a name that tied it back to this library, because names like "render-template" seemed too generic. I'll leave this open a little longer to give me time to think about a better name.